.github/workflows/release.yml

```diff
@@ -42,7 +46,29 @@ jobs:
           GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
         run: |
           echo $GPG_FILE | base64 -d > secring.gpg
-          ./gradlew publishToSonatype closeAndReleaseSonatypeStagingRepository
+          # Publish both locally and to Sonatype.
+          # The artifacts stored locally will be used to generate the SLSA provenance.
+          ./gradlew publishToMavenLocal publishToSonatype closeAndReleaseSonatypeStagingRepository
```

Comment:
Instead of publishing to maven local, which may (will) be polluted by many unrelated artifacts, you can use `publishToBuildRepository`. This will create a Maven-like repository in `$ROOT/build/repo` which will contain all the artifacts you want (plus the metadata files, pom, Gradle .module, version catalog, etc.).

Reply:
Ah sorry, this error is unrelated. It says that it won't publish a build scan on `ge.micronaut.io`, but this isn't the cause of the problem, it's because I used the wrong task name, it's `publishAllPublicationsToBuildRepository` (you can also use `pAPTBR`).

Reply:
Thanks @melix . Changed in [bf15270](https://github.com/micronaut-projects/micronaut-project-template/pull/263/commits/bf15270c64686c13a24923dd0bccac0fe1e93b59).